Buildings/Experimental/DHC/Networks/Examples/Distribution2PipeExample.mo

```diff
@@ -0,0 +1,283 @@
+  connect(volAutoSize.ports[1], pumAutoSize.port_a) annotation (Line(points={{-22,20},
+          {-8,20},{-8,30},{0,30}},     color={0,127,255}));
+  connect(THeaWatSup.y, supHeaWatAutoSize.T_in) annotation (Line(points={{-78,10},
+          {-72,10},{-72,14},{-62,14}}, color={0,0,127}));
+  connect(dpPum.y, pumAutoSize.dp_in) annotation (Line(points={{-78,40},{-60,40},
+          {-60,52},{10,52},{10,42}}, color={0,0,127}));
+  connect(reaRep.u,minTSet.y)
+    annotation (Line(points={{-62,100},{-78,100}},
+                                                color={0,0,127}));
+  connect(reaRep.y, terPlugFlow.TSetHea) annotation (Line(points={{-38,100},{30,
+          100},{30,-5},{49.1667,-5}},                 color={0,0,127}));
+  connect(reaRep1.y, terPlugFlow.QReqHea_flow) annotation (Line(points={{-38,70},
+          {26,70},{26,-11.6667},{49.1667,-11.6667}}, color={0,0,127}));
+  connect(terPlugFlow.port_bHeaWat, disPlugFlow.ports_aCon) annotation (Line(
+        points={{70,-18.3333},{74,-18.3333},{74,-60},{72,-60}},          color={
+          0,127,255}));
+  connect(disPlugFlow.ports_bCon, terPlugFlow.port_aHeaWat) annotation (Line(
+        points={{48,-60},{46,-60},{46,-18.3333},{50,-18.3333}},          color={
+          0,127,255}));
+  connect(pumPlugFlow.port_b, disPlugFlow.port_aDisSup)
+    annotation (Line(points={{20,-70},{40,-70}}, color={0,127,255}));
+  connect(volPlugFlow.ports[1], pumPlugFlow.port_a) annotation (Line(points={{-22,-80},
+          {-6,-80},{-6,-70},{0,-70}},                          color={0,127,255}));
+  connect(dpPum.y, pumPlugFlow.dp_in) annotation (Line(points={{-78,40},{-68,40},
+          {-68,-46},{10,-46},{10,-58}},                 color={0,0,127}));
+  connect(THeaWatSup.y, supHeaWatPlugFlow.T_in) annotation (Line(points={{-78,10},
+          {-74,10},{-74,-92},{-50,-92},{-50,-94}}, color={0,0,127}));
+  connect(volAutoSize.ports[2], supHeaWatAutoSize.ports[1]) annotation (Line(
+        points={{-20,20},{-22,20},{-22,9},{-40,9}}, color={0,127,255}));
+  connect(disAutoSize.port_bDisRet, supHeaWatAutoSize.ports[2]) annotation (
+      Line(points={{40,24},{40,4},{-40,4},{-40,11}}, color={0,127,255}));
+  connect(volPlugFlow.ports[2], supHeaWatPlugFlow.ports[1]) annotation (Line(
+        points={{-20,-80},{-22,-80},{-22,-99},{-28,-99}}, color={0,127,255}));
+  connect(disPlugFlow.port_bDisRet, supHeaWatPlugFlow.ports[2]) annotation (
+      Line(points={{40,-76},{40,-100},{-28,-100},{-28,-97}}, color={0,127,255}));
+  connect(pipeGroundCouplingRet.heatPorts, disPlugFlow.heatPortsRet)
+    annotation (Line(points={{53,-95},{53,-94},{52.6,-94},{52.6,-77.6}}, color=
+          {127,0,0}));
+  connect(pipeGroundCouplingSup.heatPorts, disPlugFlow.heatPortsDis)
+    annotation (Line(points={{89,-95},{89,-68.4},{67.8,-68.4}}, color={127,0,0}));
+  annotation (
+    Documentation(
+      info="<html>
+<p>
+Example model of two distribution models.  It uses 
```

Comment:
Give better description of what this example physically represents so the user doesn't have to figure it out from looking at the model alone.  Same for other distribution model and Connection Examples.

Reply:
Added description to example models